book/src/editor.md

```diff
@@ -242,6 +242,7 @@ Search specific options.
 |--|--|---------|
 | `smart-case` | Enable smart case regex searching (case-insensitive unless pattern contains upper case characters) | `true` |
 | `wrap-around`| Whether the search should wrap after depleting the matches | `true` |
+| `matches-couter-limit`| Maximum number of matches that will be counted for the denominator of `search-position`. Possible values are integers or `"none"` for no limit. | `100` |
```

Comment:
Typo:

```suggestion
| `matches-counter-limit`| Maximum number of matches that will be counted for the denominator of `search-position`. Possible values are integers or `"none"` for no limit. | `100` |
```

I know we could bikeshed this all day, so take it for what it's worth, but the option name is a bit long and it's not necessary to pluralize "matches" if you include "count". There are no other config options containing the word "limit" and several starting with "max" (`max-depth`, `max-wrap`, `max-indent-retain`), so how about `max-matches` or `max-match-count`?

Reply:
I like max-matches a lot, actually. Though I agree with you pointing out "we could bikeshed this all day" — helix tends to do that enough already. So whatever name will work, as long as it's actually accepted into core. Worst case I'll make a snarky joke about its name to myself.

Reply:
Done.